libraries/botbuilder-dialogs-adaptive/src/conditions/index.ts

```diff
@@ -5,8 +5,23 @@
  * Copyright (c) Microsoft Corporation. All rights reserved.
  * Licensed under the MIT License.
  */
-export * from './onDialogEvent';
 export * from './onCondition';
+export * from './onDialogEvent';
```

Comment:
I try to list all of the exports in alphabetical order. That way you can more easily see if you're missing an export by just comparing the exports and ensure they're in the same order as VSCode has.